src/include/daos_event.h

```diff
@@ -202,6 +202,24 @@ daos_event_next(daos_event_t *parent, daos_event_t *child);
 int
 daos_event_test(struct daos_event *ev, int64_t timeout, bool *flag);
 
+typedef int (*daos_event_comp_cb_t)(void *, daos_event_t *, int);
+
+/**
+ * Register completion callback on event.
```

Comment:
I noticed just now that our API version is still set to 0.9.0.   This should at least be a minor version bump (middle number) but might be a good time to at least go to 1.0.0 since we've released or perhaps 1.2.0 just to sync it with the daos version.   Note, the API version has nothing to do with the daos version so it can move up independently.   Regardless, a bump here will allow a user to require this version if they need this capability.

Reply:
API_VERSION is set in the SConstruct

Reply:
In general, new API means minor bump.   ABI breakage means major bump